### PR TITLE
Fix collapsing commit history

### LINKED ISSUE #661: Reword to "+X more day(s)"
Please reword this to "+X more day(s)"

_Originally posted by @joglr in https://github.com/git-truck/git-truck/pull/654#discussion_r1381650833_

### LINKED ISSUE #664: Clicking "+ X day(s) more" does not affect fold down arrow
When clicking "+ X day(s) more", the arrow right of "Commit History" should also flip.
Right now it does not, so if you expand by clicking "+ X day(s) more", and then want to fold it up using the arrow, the arrow flips but the list is not folded up. You have to click again to fold up the commit-list

image

### PR DESCRIPTION
**Pull request by tjomson**
Fixes #664 
Fixes #661

### CODE REVIEW

File: src/components/accordion/Accordion.tsx [hunk 1]
```diff
@@ -66,17 +69,17 @@ function Accordion({
...
+        toggle={() => setCollapsed(!collapsed)}
       />
     </ul>
   )
 }
 
 export function ShowMoreLabel(props: ShowMoreLabelProps) {
   if (!props.show) return null
-  return <span className="whitespace-pre text-xs font-medium opacity-70 hover:cursor-pointer" onClick={props.toggle}>+{props.items.length} day(s) more</span>
+  return <span className="whitespace-pre text-xs font-medium opacity-70 hover:cursor-pointer" onClick={props.toggle}>+{props.items.length} day{props.items.length > 1 ? "s" : ""} more</span>
```
Comment (joglr) [inline src/components/accordion/Accordion.tsx:82]: You need to switch it to "x more days" instead of "x days more" i think.

**Review by joglr — CHANGES_REQUESTED** [review]
Looks good, but needs a little change.

File: src/components/accordion/Accordion.tsx (hunk 1 above)
Comment (tjomson) [reply src/components/accordion/Accordion.tsx:82]: Ahhh, that was what you meant :D

**Review by joglr — APPROVED** [review]
LGTM

**Comment by dawidwoz** [comment]
@joglr @tjomson Good catch! Everything works smoothly now. Thank you for fixing it 🙂
